dandiapi/api/tasks/__init__.py

```diff
@@ -163,27 +162,11 @@ def delete_doi_task(doi: str) -> None:
 
 
 @shared_task
-def unembargo_dandiset(dandiset_id: int):
-    """Unembargo a dandiset by copying all embargoed asset blobs to the public bucket."""
-    dandiset: Dandiset = Dandiset.objects.get(id=dandiset_id)
-
-    # Only the draft version is needed, since embargoed dandisets can't be published
-    draft_version: Version = dandiset.draft_version
-    embargoed_assets: QuerySet[Asset] = draft_version.assets.filter(embargoed_blob__isnull=False)
-
-    # Unembargo all assets
-    for asset in embargoed_assets.iterator():
-        asset.unembargo()
```

Comment:
Is there a reason you're not removing this model method?

Reply:
Nope, thanks for catching that.